ChangeLog

```diff
@@ -17,6 +17,13 @@ Security
      implementation allowed an offline 2^80 brute force attack on the
      HMAC key of a single, uninterrupted connection (with no
      resumption of the session).
+   * Fix buffer overread vulnerability in ssl_parse_server_psk_hint
+     function. Previous implementation could in theory lead to a segmentation
+	 fault upon reading invalid input.
```

Comment:
That's a bit too wordy and not completely grammatical. Proposal:

> Fix a buffer overread in ssl_parse_server_psk_hint() that could cause a crash on invalid input.

The second one is not a security issue, so please move it to “Bugfix”. Suggested wording:

> Fix a possible arithmetic overflow in ssl_parse_server_psk_hint() that could cause a key exchange to fail on valid data.